server.js

```diff
@@ -54,8 +70,8 @@ function getForecast(weatherData) {
 // });
 
 function Location(dataFromFile, cityName) {
-  let city = Object.entries(cityName)[0][1];
-  this.search_query = city;
+  //let city = Object.entries(cityName)[0][1];
```

Comment:
Why let vs const? I'm not completely clear on when to use either, aside from being told to use const recently